Paper.tex

```diff
@@ -1386,33 +1453,34 @@ \section{Precompiled Contracts}\label{app:precompiled}
 Finally, the fourth contract, the identity function $\Xi_{\mathtt{ID}}$ simply defines the output as the input:
 \begin{eqnarray}
 \Xi_{\mathtt{ID}} &\equiv& \Xi_{\mathtt{PRE}} \quad \text{where:} \\
-g_r &=& 15 + 3\Big\lceil \dfrac{|I_\mathbf{d}|}{32} \Big\rceil\\
-\mathbf{o} &=& I_\mathbf{d}
+\mathbf{g}_{\mathrm{r}} &=& 15 + 3\Big\lceil \dfrac{|I_{\mathrm{d}}|}{32} \Big\rceil\\
+\mathbf{o} &=& I_{\mathrm{d}}
```

Comment:
The above have already been merged #605, except with \mathrm{g} and I_{mathbf{d}} is unchanged.